api/app/Enums/AssessmentDecision.php

```diff
@@ -2,9 +2,18 @@
 
 namespace App\Enums;
 
+use App\Traits\HasLocalization;
+
 enum AssessmentDecision
 {
+    use HasLocalization;
+
     case HOLD;
     case SUCCESSFUL;
     case UNSUCCESSFUL;
+
+    public static function getLangFilename(): string
+    {
+        return 'assessment_decision';
```

Comment:
missing file?

Reply:
Added in https://github.com/GCTC-NTGC/gc-digital-talent/pull/10701/commits/e0143f0d1256e92bf43756ccdcb5397bdc95b781